test/util/test_helpers.py

```diff
@@ -161,3 +181,37 @@ def validate_netrc_contents(self):
         self.assertEquals(lines[0].strip(), "machine https://someurl")
         self.assertEquals(lines[1].strip(), "login somelogin")
         self.assertEquals(lines[2].strip(), "password somepass")
+
+
+def get_credentials() -> CredentialsData:
```

Comment:
these are under UNIT TEST HELPERS where they should be under INTEGRATION TEST HELPERS under CREDENTIAL STORE (like ***init*** there, etc)

Reply:
done